holidays/countries/united_states.py

```
@@ -33,56 +33,62 @@
-        - PA: https://www.legis.state.pa.us/WU01/LI/LI/US/PDF/1893/0/0138..PDF
-        - PR: https://en.wikipedia.org/wiki/Public_holidays_in_Puerto_Rico
-        - SC: https://www.scstatehouse.gov/code/t53c005.php
-        - TN: https://www.tn.gov/about-tn/state-holidays.html
-        - TX: https://www.tsl.texas.gov/ref/abouttx/holidays
-        - UT: https://le.utah.gov/xcode/Title63G/Chapter1/63G-1-S301.html
-        - VA: https://law.lis.virginia.gov/vacode/title2.2/chapter33/section2.2-3300/
-        - VT: https://legislature.vermont.gov/statutes/section/01/007/00371
-        - WA: https://app.leg.wa.gov/rcw/default.aspx?cite=1.16.050
-        - WV: https://code.wvlegislature.gov/2-2-1/
-        - WY: https://ai.wyo.gov/about-us/state-holidays-and-office-closures
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)